ts/__e2e__/messages.e2e.ts

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
I'm divided about ignoring the entire file. At this point it may just become`.js`.

Reply:
This was required for the impossibility to import `import { device } from "detox";`. 
I went a little deeper and I found the file `ts/@types/detox.d.ts` that was an ovverride of the default types exported by the `detox` package. Removed in https://github.com/pagopa/io-app/pull/3482/commits/7de1f051329bd5bac3009e71515596b2025cb043 and also removed the, now, unnecessary:
```
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```